test/jest/roktManager.spec.ts

```diff
@@ -23,6 +23,129 @@ describe('RoktManager', () => {
         });
     });
 
+    describe('#hashAttributes', () => {
+        beforeEach(() => {
+            roktManager['currentUser'] = currentUser;
+        });
+
+        it('should call kit.hashAttributes with empty attributes', () => {
```

Comment:
nit:

seems like an AI test that isn't relevant since the test directly below it is doing the same thing...just that in this test it's an empty object that's passed in and spit back out.

Reply:
My intention was to be explicit with the use cases of an empty object vs a populated object, especially if we decide in the future to return something other than an empty object, i.e. `null` or throw an error.